encodings/fastlanes/src/bitpacking/compute/search_sorted.rs

```diff
@@ -158,14 +157,7 @@ impl<'a, T: BitPacking + NativePType> BitPackedSearch<'a, T> {
             .map(|p| p.min_index())
             .transpose()?
             .unwrap_or_else(|| array.len());
-        let first_non_null_idx = match array.validity_mask()? {
-            Mask::AllTrue(_) => 0,
-            Mask::AllFalse(n) => n,
-            Mask::Values(v) => {
-                // In sorted order, nulls come before all the non-null values, i.e. we have true count worth of non null values at the end
-                array.len() - v.true_count()
-            }
-        };
+        let first_non_null_idx = array.validity().null_count(array.len())?;
```

Comment:
I meant `array.invalid_count()?` :D But this works too

Reply:
invalid_count is a thing on encoding, null_count is a method on array